infrastructure/src/main/java/org/corfudb/infrastructure/ManagementServer.java

```diff
@@ -113,15 +114,16 @@ public ExecutorService getExecutor() {
      */
     public ManagementServer(ServerContext serverContext) {
 
-        this.opts = serverContext.getServerConfig();
-        this.localEndpoint = this.opts.get("--address") + ":" + this.opts.get("<port>");
+        managementServerConfig = ManagementServerConfig.parse(serverContext.getServerConfig());
+
```

Comment:
Server context already has a method to getLocalEndpoint. You can use that directly.

Reply:
yes, I think it is much cleaner to move the responsibility to ServerContext. Moved all of them there.

Reply:
done